src/handlers/rrp.ts

```diff
@@ -165,6 +164,7 @@ export const handler: ScheduledHandler = async (event: ScheduledEvent, context:
         });
         if (!currentBlock.success) {
           utils.logger.error('failed to fetch the blockNumber', currentBlock.error);
+          utils.removeMetadata(['Chain-ID', 'Provider']);
```

Comment:
Indeed it is a bit weird. I guess it would be better to call it in the `logger` object like this: `utils.logger.removeMetadata(...)`.

`utils` is the whole `airnode-utilities` package, are you suggesting to not use the `utils` import alias and just import the function from the package?

Reply:
I was hoping that we can rename the import but now I see that we're using other nonlogging-related functions from the package. Never mind then, it's fine as it is.